app/stylesheets/layout.scss

```diff
@@ -30,7 +30,7 @@ body {
   left: 0;
   position: absolute;
   right: 0;
-  top: $header-height + $filters-closed-height;
+  top: $header-height;
```

Comment:
it's subtle: without it, the map extends behind the filters, end then calling `map.fitBounds` makes the layers we show go behind the filters.